openpype/hosts/photoshop/api/extension/index.html

```diff
@@ -102,16 +113,17 @@
   <script type="text/javascript" src="./client/wsrpc.js"></script>
   <script type="text/javascript" src="./client/CSInterface.js"></script>
   <script type="text/javascript" src="./client/loglevel.min.js"></script>
-  
+
   <!-- helper library for better debugging of .jsx check its license! -->
   <script type="text/javascript" src="./host/JSX.js"></script>
-  
+
   <script type="text/javascript" src="./client/client.js"></script>
-  
+
     <a href=# id=workfiles-button><button>Workfiles...</button></a>
     <a href=# id=creator-button><button>Create...</button></a>
     <a href=# id=loader-button><button>Load...</button></a>
     <a href=# id=publish-button><button>Publish...</button></a>
+    <a href=# id=studio-tools-button><button>Studio Tools...</button></a>
```

Comment:
```suggestion
    <a href=# id=sceneinventory-button><button>Manage...</button></a>
    <a href=# id=subsetmanager-button><button>Subset Manager...</button></a>
    <a href=# id=studio-tools-button><button>Studio Tools...</button></a>
    <a href=# id=experimental-button><button>Experimental Tools...</button></a>
```
I would move it a bit lower as for majority of studios it will be empty, I guess.